ibft/instance/commit.go

```diff
@@ -65,12 +76,17 @@ func (i *Instance) CommitMsgValidationPipeline() pipeline.Pipeline {
 
 // CommitMsgValidationPipelineV0 is version 0
 func (i *Instance) CommitMsgValidationPipelineV0() pipeline.Pipeline {
+	return CommitMsgValidationPipelineV0(i.State().Lambda.Get(), i.State().SeqNumber.Get(), i.ValidatorShare)
+}
+
+// CommitMsgValidationPipelineV0 is version 0 of commit message validation
+func CommitMsgValidationPipelineV0(identifier []byte, seq uint64, share *storage.Share) pipeline.Pipeline {
```

Comment:
that seems redundant ..

Reply:
its a function so the exporter could use it as well

Reply:
we have the instance which has that function why create another?

Reply:
late commit is done on controller level when the instance finished and on exporter, in both cases we don't have an instance..